test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt

```diff
@@ -0,0 +1,14 @@
+
+
+You must [check with the Home Office](https://eforms.homeoffice.gov.uk/outreach/lcs-application.ofml) to find out if the tenant is allowed to rent property. You’ll need the person’s Home Office reference number.
+
+You’ll get an answer within 2 working days.
+
+If the person’s allowed to rent, the Home Office will tell you if you need to check their documents again later.
```

Comment:
@leenagupte here d0f2b65e17b61d9345605e5237b6de8b999b3422 and 94bbd03c49f76c0aaaa8573b94144dc41c8fcd92